### ISSUE TITLE
At-a-glance overview of live nodes & their stats

### ISSUE THREAD

**Issue by rafaeldjpbrochado** [iotaledger/explorer#49]
## Description

It'd be great if the Tangle Explorer homepage told us how many nodes are live in the network at any given time, akin to the [Helium explorer](https://explorer.helium.com/coverage).

Though geographical coverage is not a necessity in IOTA, I find it would still be nice to have an at-a-glance view (e.g. world map view) clearly indicating:

- How many nodes are live in the network
- Which nodes are permanodes, and a total permanode count
- Which nodes are Bee/Hornet/Wasp, and a total count for each
- Where nodes are located (preferably just countries, no cities) and who they're peered to
- How much mana does each node have (post-Coordicide)
- For how long each node has been online
- Etc

## Motivation

I don't think there's enough transparency around such stats, yet. Plus, a visualization like this would let us see the network expand or contract in real-time, which could then further inform us about the network's incentive structure.

For example, I think it would be interesting to study and compare the historical evolution of the chrysalis and coordicide networks (no mana vs. mana) from the moment they go live on mainnet. The current hypothesis is that network access is enough incentive for community members to spin up IOTA nodes. But is it really? I'd love to track that data somehow & then analyze it. Will we see a 10x increase in node operators when coordicide goes live? We won't know unless we track it.

These kinds of stats could, for example, reveal that the number of mainnet node operators was steadily declining over an extended period of time. This would in turn put some pressure on us to assess the issue & rethink our incentive structure such that more people feel compelled to spin up a node and thus maximize decentralization.

## Open questions & Security concerns

I don't know what would qualify as sensitive information here, but I guess we wouldn't have to display stuff like hardware specs, node software version, identifiable info, etc.

I'm sure we could come up with ways to minimize the associated risks (if any) whilst maximizing network transparency & data collection that we can later leverage to make better informed decisions as a community.

**Comment by JanOonk** [comment]
-Distribution of versions between nodes i.e. how many nodes (and as percentage) run version X  
-Graphs over time for cumulative number of messages (per payload type), number of online nodes, mps  
-Piechart showing percentage of each payload type  
-Is there a way to identify which client/device sent the transaction? Would be cool if you can see in a graph or statistic that a certain dApp, project or company is using the Tangle more and more over time.
-For specific payload type like value transactions you could even show more details like a graph over time with total value sent per day  
-...  

Basically any stats and graphs which shows adoption/usage would be cool. And all this on a easy to use dashboard.
I recommend using Javascript library [Chart.js](https://www.chartjs.org/) for the graphs.

**Comment by rafaeldjpbrochado** [comment]
@obany: Is there a repo for the v2.iota.org explorer/visualizer? Do you know if this old explorer will be abandoned in the future in favor of the approach used at v2.iota.org? I quite like that integrated approach btw, where you land on IOTA's main website and immediately you can see the explorer/visualizer without ever leaving the main website.

@JanOonk: Great stat suggestions!

**Comment by obany** [comment]
@rafaeldjpbrochado There is some work planned to update the explorer, but it is in its early stages.

**Comment by rafaeldjpbrochado** [comment]
I changed my mind about hardware specs. It would be nice indeed to get an overview of all the hardware that's being used to keep the network online. This could inform us about the network's real-time energy consumption, for example.
